src/main/resources/locales/cy/translation/acas-cert-num.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "Acas certificate number - Requires Update",
+  "acasCertNum": "Enter an Acas early conciliation certificate number - Requires Update",
+  "h1": "A oes gennych rif tystysgrif ACAS ar gyfer ",
+  "hintText": "Nearly everyone should have this number before they fill in a claim form. You can find it on your Acas certificate. Read more about the - Requires Update",
+  "linkText": "Acas early conciliation service(opens in new tab) - Requires Update",
+  "yes": "Oes",
```

Comment:
For me is unclear when which to use, because Yes or No is differs in different places...

Reply:
It seems like 'YES' from this file is not being used/read but agree we should use YES and NO from the Common translations.